src/esbmc/bmc.cpp

```diff
@@ -175,10 +195,13 @@ smt_convt::resultt bmct::run_decision_procedure(
 
   std::stringstream ss;
   ss << "Solving with solver " << smt_conv->solver_text();
+  std::thread t1(printAliveMessage, msg);
```

Comment:
Name it with something like `print_alive`. Also this thread should only happen if the user calls for it (add an option).